### ISSUE TITLE
compat: A5E for DnD5e

### ISSUE THREAD

**Issue by kgar** [kgar/foundry-vtt-tidy-5e-sheets#580]
Determine if Tidy 5e and a5e-for-dnd5e can establish compatibility
https://github.com/JPMeehan/a5e-for-dnd5e

https://discord.com/channels/1167985253072257115/1233621696943230976

**Comment by JPMeehan** [comment]
Tracking on my repository as well: https://github.com/JPMeehan/a5e-for-dnd5e/issues/23

**Comment by kgar** [comment]
I'm letting this one go. Too much has changed, and it looks like the module is not going to be in newer dnd5e versions.

**Comment by JPMeehan** [comment]
Yeah I just don't have time these days to maintain a module I'm not really using

**Comment by kgar** [comment]
> Yeah I just don't have time these days to maintain a module I'm not really using

Totally understandable! If I can assist with integration for current projects, my dev schedule will open up some in another few months.
